.golangci.yml

```diff
@@ -27,6 +27,7 @@ linters:
     - gomnd     # too annoying
     - lll       # gRPC types are long
     - wsl       # too annoying
+    - unused    # too many false positives
```

Comment:
It actually doesn't have false positives: we don't use that code outside of the tests, and linter is doing a good job checking that we don't keep unused code.

Let's keep it there, but I will revert that change after merging.